scripts/typescript/normalize-import.spec.js

```diff
@@ -64,9 +64,9 @@ describe(`normalize-import`, () => {
   it(`should throw error if provided --output is not a valid directory`, () => {
     const utils = setup();
 
-    expect(() => execSync(`node ${utils.scriptPath} --output some/nonexistent/path`, { stdio: 'pipe' })).toThrowError(
-      `some/nonexistent/path doesn't exist`,
-    );
+    expect(() =>
+      execSync(`node ${utils.scriptPath} --output /some[\\/]nonexistent[\\/]path doesn't exist/`, { stdio: 'pipe' }),
+    ).toThrowError(`/some[\\/]nonexistent[\\/]path doesn't exist/`);
```

Comment:
This should be a regex object (no quotes)
```suggestion
    ).toThrowError(/some[\\/]nonexistent[\\/]path doesn't exist/);
```